server/src/main/java/de/uftos/services/TimetableService.java

```diff
@@ -168,10 +175,22 @@ public Timetable getById(String id) {
    */
   public Timetable create(TimetableRequestDto timetable) throws ResponseStatusException {
     if (timetable.name().isBlank()) {
-      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "The name of the timetable is blank.");
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST,
+          "The name of the timetable is blank.");
     }
     Consumer<TimetableSolutionDto> solverFinishedEvent = (solution) -> {
+      if (solution.hardScore() > 0) {
+        this.notificationRepository.notify(NotificationType.EMAIL,
+            "Stundenplan konnte nicht erstellt werden",
+            "%d Hard Constraint(s) und %d Soft Constraint(s) konnten nicht erfüllt werden".formatted(
+                solution.hardScore(), solution.softScore()));
```

Comment:
These values need to be inverted as well:
```suggestion
                -solution.hardScore(), -solution.softScore()));
```